symbols/Espressif.kicad_sym

```diff
@@ -2392,6 +2392,175 @@
+        (name "MTDI/GPIO12/ADC2_CH5" (effects (font (size 1.27 1.27))))
+        (number "18" (effects (font (size 1.27 1.27))))
+      )
+      (pin bidirectional line (at -33.02 -6.35 0) (length 2.54)
+        (name "MTCK/GPIO13/ADC2_CH4" (effects (font (size 1.27 1.27))))
+        (number "19" (effects (font (size 1.27 1.27))))
+      )
+      (pin power_in line (at 2.54 35.56 270) (length 2.54)
+        (name "3V3" (effects (font (size 1.27 1.27))))
+        (number "2" (effects (font (size 1.27 1.27))))
+      )
+      (pin bidirectional line (at -33.02 -8.89 0) (length 2.54)
+        (name "MTDO/GPIO15/ADC2_CH3" (effects (font (size 1.27 1.27))))
+        (number "20" (effects (font (size 1.27 1.27))))
+      )
+      (pin bidirectional line (at 33.02 21.59 180) (length 2.54)
+        (name "ADC2_CH2/GPIO2" (effects (font (size 1.27 1.27))))
+        (number "21" (effects (font (size 1.27 1.27))))
+      )
+      (pin bidirectional line (at 33.02 24.13 180) (length 2.54)
+        (name "GPIO0/BOOT/ADC2_CH1" (effects (font (size 1.27 1.27))))
+        (number "22" (effects (font (size 1.27 1.27))))
+      )
+      (pin bidirectional line (at 33.02 19.05 180) (length 2.54)
+        (name "ADC2_CH0/GPIO4" (effects (font (size 1.27 1.27))))
+        (number "23" (effects (font (size 1.27 1.27))))
+      )
+      (pin bidirectional line (at 33.02 15.24 180) (length 2.54)
+        (name "SD_DATA2/GPIO9" (effects (font (size 1.27 1.27))))
+        (number "24" (effects (font (size 1.27 1.27))))
+      )
+      (pin bidirectional line (at 33.02 12.7 180) (length 2.54)
+        (name "SD_DATA3/GPIO10" (effects (font (size 1.27 1.27))))
+        (number "25" (effects (font (size 1.27 1.27))))
+      )
+      (pin bidirectional line (at 33.02 8.89 180) (length 2.54)
+        (name "GPIO5" (effects (font (size 1.27 1.27))))
+        (number "26" (effects (font (size 1.27 1.27))))
+      )
+      (pin bidirectional line (at 33.02 6.35 180) (length 2.54)
+        (name "GPIO18" (effects (font (size 1.27 1.27))))
+        (number "27" (effects (font (size 1.27 1.27))))
+      )
+      (pin bidirectional line (at 33.02 3.81 180) (length 2.54)
+        (name "GPIO23" (effects (font (size 1.27 1.27))))
+        (number "28" (effects (font (size 1.27 1.27))))
+      )
+      (pin bidirectional line (at 33.02 1.27 180) (length 2.54)
+        (name "GPIO19" (effects (font (size 1.27 1.27))))
+        (number "29" (effects (font (size 1.27 1.27))))
+      )
+      (pin bidirectional line (at 33.02 -3.81 180) (length 2.54)
+        (name "GPIO22" (effects (font (size 1.27 1.27))))
+        (number "30" (effects (font (size 1.27 1.27))))
+      )
+      (pin bidirectional line (at 33.02 -1.27 180) (length 2.54)
+        (name "GPIO21" (effects (font (size 1.27 1.27))))
+        (number "31" (effects (font (size 1.27 1.27))))
+      )
+      (pin bidirectional line (at 33.02 27.94 180) (length 2.54)
+        (name "U0RXD/GPIO3" (effects (font (size 1.27 1.27))))
```

Comment:
There is an inconsistency in our docs.
Please change the pins:

- Pin 32 is U0TX/GPIO1
- Pin 33 is U0RX/GPIO3

The official Espressif documentation will be updated/fixed.